lib/actions/environment-variables.ts

```diff
@@ -1,154 +0,0 @@
-/*
```

Comment:
This file implemented `balena envs` and `balena env rename` with Capitano. This PR migrates those commands to oclif.